hooks/declarative-subsequent-scans/templates/NOTES.txt

```diff
@@ -0,0 +1,13 @@
+Declarative Subsequent Scan Hook deployed.
+
+This will allow you to start Scans based on previous findings.
+E.g. start a ssh scan for every open ssh port found.
+
+The rules are defined as kubernetes CRD's. You can list all of these `CascadingScanRules` by running:
+
+$ kubectl get cascadingrules
+
+You need to explicitly turn on scan cascading for every scan you use.
+You can do that by setting a label selector which matches all rules you want to use.
+
+Find out more, on the docs: TODO
```

Comment:
Unfinished TODO without assigend ticket.

Reply:
Issue created and linked